.github/CODEOWNERS

```
@@ -0,0 +1,7 @@
+# Techical Writer Review
+
+/website/content/docs/* @hashicorp/consul-docs
+/website/content/commands/* @hashicorp/consul-docs
+/website/content/api-docs/* @hashicorp/consul-docs
```

Reading https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners, it sounds like this `*` only matches files in that directory, not all sub-directories.

We appear to have many subdirectories, so these patterns won't match most files.

I believe the `*` needs to be removed

---

```suggestion
/website/content/docs/ @hashicorp/consul-docs
/website/content/commands/ @hashicorp/consul-docs
/website/content/api-docs/ @hashicorp/consul-docs
```

@dnephin like this ^

---

That's my understanding yes, I think that should fix it